mlflow/pyfunc/__init__.py

```diff
@@ -2429,12 +2429,16 @@ def predict(model_input: List[str]) -> List[str]:
                 CHAT_MODEL_INPUT_SCHEMA,
                 CHAT_MODEL_OUTPUT_SCHEMA,
             )
-            input_example = CHAT_MODEL_INPUT_EXAMPLE
+            input_example = input_example or CHAT_MODEL_INPUT_EXAMPLE
 
-            # perform output validation and throw if
-            # output is not coercable to ChatResponse
-            messages = [ChatMessage(**m) for m in input_example["messages"]]
-            params = ChatParams(**{k: v for k, v in input_example.items() if k != "messages"})
+            if isinstance(input_example, list) and isinstance(input_example[0], ChatMessage):
```

Comment:
@sunishsheth2009 Could you verify passing a list of ChatMessage as input example works on model endpoint? I wonder how it will be rendered as input example in UI, and if that passes schema validation when queried.

Reply:
I think it gets converted to json. Added a test for that